frontend/js/pong/displayCharacter.js

```diff
@@ -39,4 +43,13 @@ async function displayCharacter(player ,environment, color, name) {
 	return (player);
 }
 
-export { displayCharacter };
\ No newline at end of file
+function updateMixers(player1, player2) {
+	const delta = clock.getDelta();
+
+	if (player1.character.mixer && player2.character.mixer) {
+		player1.character.mixer.update(delta);
```

Comment:
Here you can use `player1.character?.mixer`
or 
`if (!player1.character.mixer && !player2.character.mixer) return;`